src/main/java/com/cloudbees/plugins/credentials/CredentialsProvider.java

```diff
@@ -1075,7 +1075,7 @@ public String getDisplayName() {
      */
     @Override
     public String getIconClassName() {
-        return "icon-credentials-credentials";
+        return "symbol-key";
```

Comment:
I think the `wallet` icon would have been a better choice for the reasons explained in [JENKINS-69775](https://issues.jenkins.io/browse/JENKINS-69775).